nautobot/docs/user-guides/ip-address-merge-tool.md

```diff
@@ -0,0 +1,65 @@
+
+## When to use this tool
+
+After upgrading to Nautobot v2.0 and running the data migrations necessary, duplicate `IPAddress` objects might exist in your database. We define duplicate `IPAddress` objects as those which have the same `host` attribute but exist in different `namespaces`. If you have no use case to keep those duplicate `IPAddress` objects around, we recommend you to use this tool to de-duplicate those `IPAddress` objects and keep your database clean and manageable. But if you do have reasons to maintain duplicate `IPAddress` objects, this tool is not for you.
+
+!!! important
+    One of the possible reasons to maintain duplicate `IPAddress` objects can be the following:
+    1. You have use cases for duplicate `IPAddress` objects with different `nat_inside` attributes. Only one `nat_inside` IP address can be assigned per object.
+
+## How to access this tool
+
+To use this tool:
+
+1. Go to the `IPAM` tab on the navigation menu and click on `IP Addresses`. This will take you to the list view of all `IPAddress` objects that exist in the database.
+2. On the top right of the page, you will find a group of buttons representing different available actions and the first one should be `Find and Merge Duplicate IPs`.
+3. Click on that button to access the tool.
+
+![Homepage Navigation](./images/ip-address-merge-tool/navigation.png)
+
+![Merge Button](./images/ip-address-merge-tool/ip_merge_button.png)
+
+## How it works
+
+1. All of the selected `IPAddress` objects will be deleted and a new `IPAddress` object with all of the desired attributes will be created.
+2. If you de-select all other duplicates and attempt to merge, nothing will happen (the operation is a no-op) and the object will be skipped over as if you had clicked the `Skip and Go to the Next Duplicate` button.
+3. All `Interface` assignments of the deleted `IPAddress` objects will be automatically updated to reference the newly created `IPAddress` object.
+4. All `VMInterface` assignments of the deleted `IPAddress` objects will be automatically updated to reference the newly created `IPAddress` object.
+5. The newly created `IPAddress` object will be added to `IPAddress` assignments of related `Service` objects.
+6. `primary_ip4/primary_ip6` of `Devices` that are referencing the deleted `IPAddress` objects will be automatically updated to reference the newly created `IPAddress` object.
+
+## How to use this tool
+
+Clicking on the `Find and Merge Duplicate IPs` button will automatically query your database for duplicate `IPAddress` objects and group them by their respective `host` values. The tool will present the duplicate `IPAddress` objects in order from lowest to highest `mask_length` values.
+
+![IP Address Merge View](./images/ip-address-merge-tool/merge_view.png)
+
+### Merging All Duplicate IP Addresses Presented with Desired Attributes
```

Comment:
Let's make these headings a bit more succinct? Plus be consistent at least within this page about sentence case vs title case?

```suggestion
### Merging all presented IP addresses
```